test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -83,15 +83,23 @@ class test_ucp_sockaddr : public ucp_test {
                         ucs_log_level_t level, const char *message, va_list ap)
     {
         if (level == UCS_LOG_LEVEL_ERROR) {
-            std::string err_str = format_message(message, ap);
-            if ((strstr(err_str.c_str(), "no supported sockaddr auxiliary transports found for")) ||
-                (strstr(err_str.c_str(), "sockaddr aux resources addresses")) ||
-                (strstr(err_str.c_str(), "no peer failure handler")) ||
-                (strstr(err_str.c_str(), "connection request failed on listener")) ||
+            static std::vector<std::string> stop_list;
+            if (stop_list.empty()) {
+                stop_list.push_back("no supported sockaddr auxiliary transports found for");
+                stop_list.push_back("sockaddr aux resources addresses");
+                stop_list.push_back("no peer failure handler");
+                stop_list.push_back("connection request failed on listener");
                 /* when the "peer failure" error happens, it is followed by: */
-                (strstr(err_str.c_str(), "received event RDMA_CM_EVENT_UNREACHABLE"))) {
-                UCS_TEST_MESSAGE << err_str;
-                return UCS_LOG_FUNC_RC_STOP;
+                stop_list.push_back("received event RDMA_CM_EVENT_UNREACHABLE");
+                stop_list.push_back(ucs_status_string(UCS_ERR_UNREACHABLE));
```

Comment:
when is UCS_ERR_UNREACHABLE returned?

Reply:
from pack_cb in new CM wireup protocol which is disabled for now.